code/modules/mob/living/living.dm

```diff
@@ -157,9 +153,7 @@
 /mob/living/proc/PushAM(atom/movable/AM, force = move_force)
 
 	if(isstructure(AM) && AM.pulledby)
-		if(a_intent == INTENT_HELP && AM.pulledby != src) // Help intent doesn't push other peoples pulled structures
-			return FALSE
-		if(get_dist(get_step(AM, get_dir(src, AM)), AM.pulledby)>1)//Release pulled structures beyond 1 distance
+		if(get_dist(get_step(AM, get_dir(src, AM)), AM.pulledby) > 1) //Release pulled structures beyond 1 distance
```

Comment:
This is kinda janky, but I'd rather address this if/when I update pulling, in general, so it gets to live *for now*.